lib/theme.js

```diff
@@ -37,6 +37,7 @@ export default {
         text: '#D7C9EA',
         textMuted: '#7E7E88',
         textSecondary: '#7E7E88',
+        borderGrey: '#D7C9EA',
```

Comment:
An example of why using the name of a color for a color token isn't recommended. The light-mode color value #D4D9E1 has an analogue called [secondaryMuted](https://github.com/makerdao/dai-ui/blob/master/packages/dai-ui-theme-maker-neue/lib/index.js#L22) which could be used in place of `borderGrey`. Not a huge priority to update that at this time though, just something to consider when we start thinking about refreshing the UI down the road.